subsys/net/lib/nrf_cloud/src/nrf_cloud.c

```diff
@@ -20,7 +20,7 @@ LOG_MODULE_REGISTER(nrf_cloud, CONFIG_NRF_CLOUD_LOG_LEVEL);
 #define NOT_VALID_STATE(EXPECTED) ((EXPECTED) < m_current_state)
 
 /* Identifier for user association message sent to the cloud.
- * A unique, unsigned 16 bit magic number. Can't be zero.
+ * A unique, unsigned 16p bit magic number. Can't be zero.
```

Comment:
Was that change intended? If so what does that mean?

Reply:
Definitely not intended. Will fix.